sssom/cli.py

```diff
@@ -31,51 +52,86 @@ def main(verbose, quiet):
 
 
 @main.command()
-@click.option('-i', '--input')
-@click.option('-f', '--format')
-@click.option('-o', '--output')
-@click.option('-t', '--to-format')
-@click.option('-c', '--context')
+@click.option('-i', '--input', help=help_input)
+@click.option('-f', '--format', help=help_input_format)
+@click.option('-o', '--output', help= help_output)
+@click.option('-t', '--to-format', help=help_format)
+@click.option('-c', '--context', help=help_context)
 def convert(input: str, output: str, format: str, to_format: str, context: str):
-    """
-    convert file (currently only supports conversion to RDF)
-    """
+    """Convert file (currently only supports conversion to RDF)
+
+    Args:
+
+        input (str): Input filename.
+
+        output (str): Output filename.
+
+        format (str): Input format.
+
+        to_format (str): Output Format.
+
+        context (str): Context.
+    """   
+
     convert_file(input=input, output=output, input_format=format, output_format=to_format, context_path=context)
 
 ## Input and metadata would be files (file paths). Check if exists.
 @main.command()
-@click.option('-i', '--input', required=True, type=click.Path())
+@click.option('-i', '--input', required=True, type=click.Path(), help=help_input)
 @click.option('-I', '--input-format', required=False,
-              type=click.Choice(SSSOM_READ_FORMATS, case_sensitive=False))
-@click.option('-m', '--metadata', required=False, type=click.Path())
+              type=click.Choice(SSSOM_READ_FORMATS, case_sensitive=False), help=help_input_format)
+@click.option('-m', '--metadata', required=False, type=click.Path(), help= help_metadata)
 @click.option('-c', '--curie-map-mode', default='metadata_only', show_default=True, required=True,
-              type=click.Choice(['metadata_only', 'sssom_default_only', 'merged'], case_sensitive=False))
-@click.option('-o', '--output', type=click.Path())
+              type=click.Choice(['metadata_only', 'sssom_default_only', 'merged'], case_sensitive=False), help=help_curie_map_mode)
+@click.option('-o', '--output', type=click.Path(), help= help_output)
 def parse(input: str, input_format: str, metadata:str, curie_map_mode: str, output: str):
-    """
-    parse file (currently only supports conversion to RDF)
-    """
+    """parse file (currently only supports conversion to RDF)
+
+    Args:
+
+        input (str): Input filename.
```

Comment:
this is redundant with what is declared above. Is it necessary?

Args will be shown anyway by click if you say `--help`

I think it is better to use this space to give examples, etc